third_party/mkl-dnn.BUILD

```diff
@@ -122,4 +126,7 @@ cc_library(
         "@pytorch//tools/config:thread_sanitizer": [],
         "//conditions:default": ["@tbb"],
     }),
+    defines = [
```

Comment:
Hi @jgong5 We found that bazel test fails if we do not define this macro. We also found that the failure can be fixed alternatively by adding `#cmakedefine01 DNNL_ENABLE_MAX_CPU_ISA` to file `oneDNN/include/oneapi/dnnl/dnnl_config.h.in` and adding a rule like `"#cmakedefine01 DNNL_ENABLE_MAX_CPU_ISA": "#define DNNL_ENABLE_MAX_CPU_ISA1"` in this file.
Which do you think is better?

Reply:
Let's add the definition here. Please also check and add all the default on macros here.

Reply:
OK. I have added other macro definitions here as well.